src/Misc/PerformanceViewer/performanceViewerCollectionStrategies.ts

```diff
@@ -237,18 +240,29 @@ export class PerfCollectionStrategy {
     public static AbsoluteFpsStrategy(): PerfStrategyInitialization {
         return (scene) => {
             let startTime = PrecisionDate.Now;
-            let timeTaken = 1;
+            let timeTaken = 0;
+            let frameCount = 0;
             const onBeforeAnimationsObserver = scene.onBeforeAnimationsObservable.add(() => {
+                if (frameCount === _absoluteFpsAccFrames) {
+                    frameCount = 0;
+                    timeTaken = 0;
+                }
                 startTime = PrecisionDate.Now;
             });
 
             const onAfterRenderObserver = scene.onAfterRenderObservable.add(() => {
-                timeTaken = PrecisionDate.Now - startTime;
+                timeTaken += PrecisionDate.Now - startTime;
+                frameCount += 1;
             });
 
             return {
                 id: "Absolute FPS",
-                getData: () => 1000.0 / timeTaken,
+                getData: () => {
+                    if (timeTaken < _absoluteFpsAccuracy) {
+                        timeTaken = _absoluteFpsAccuracy;
+                    }
+                    return (_absoluteFpsAccFrames * 1000.0) / timeTaken

```

Comment:
why not using the average value from the perf metric? The PerfCounter class is building it for you

Reply:
Updated to use SceneInstrumentation! Thanks for the review :D